.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,11 +8,10 @@
 [ ] Add a CLI option
 [ ] Add something to the core
 [ ] Other, please explain:
-```
-
-*If the item you've check above has a template, please paste the template questions here and answer them.*
-
 
+<!--
+    If the item you've check above has a template, please paste the template questions here and answer them.
```

Comment:
Was going to raise this as a separate issue but I'll do it here since you're looking at this file: Should we also allow folks to just reference the issue, if the issue already answered the appropriate questions?